### ISSUE TITLE
Ueberzug hangs when loading an image

### ISSUE THREAD

**Issue by Co3co** [jstkdng/ueberzugpp#135]
Ueberzug(pp) processes hang when trying to load an image. I have tried both `ueberzug` and `ueberzugpp`, though I am not sure what the differences are between them.
Log: [ueberzugpp-coco.log](https://github.com/jstkdng/ueberzugpp/files/12719364/ueberzugpp-coco.log)

```
coco@coco ~ $ ueberzugpp layer
^Cmemory: high-water mark 0 bytes
coco@coco ~ $ ueberzugpp layer
{"action":"add","identifier":"PREVIEW","max_height":49,"max_width":102,"path":"/home/coco/Desktop/temp/test/C.jpg","x":105,"y":2}
^C^C^Czsh: killed     ueberzugpp layer
```
Ctrl-C did not work in the second command. I had to send a SIGKILL signal to stop the `ueberzugpp` process.
The first time I ran `ueberzugpp layer`, I got this error:
`(process:32340): VIPS-WARNING **: 17:16:41.532: unable to load "/usr/lib/vips-modules-8.14/vips-openslide.so" -- libopenslide.so.0: cannot open shared object file: No such file or directory`
Installing the `openslide` package fixed this. I don't think this is relevant to the issue, but I figured more information wouldn't hurt.

Terminals used: [st](https://st.suckless.org/) built with [sixel support](https://gist.github.com/saitoha/70e0fdf22e3e8f63ce937c7f7da71809), kitty
Shell: zsh 5.9
Window Manager/Display Server: river (wlroots Wayland compositor)

**Comment by Co3co** [comment]
Another thing I should note, the hung `ueberzugpp` processes are using a large amount of memory (over 170MB) for images that aren't even over 1MB.

**Comment by jstkdng** [comment]
can you post the image?

**Comment by Co3co** [comment]
It happens on every image/video thumbnail/gif/PDF I try, but here is the one I was using in the examples.
![C](https://github.com/jstkdng/ueberzugpp/assets/84296063/31114875-060e-481c-aeb6-422cb7dc636b)

**Comment by jstkdng** [comment]
That's a cute monero-chan

I see the issue, `ueberzugpp` can't automatically detect the output backend to use. Can you try to run `ueberzugpp` with the `-o` flag?

as in, `ueberzugpp layer -o kitty` or `ueberzugpp layer -o sixel`

**Comment by Co3co** [comment]
Sorry for the late replies, I've been overwhelmed with stuff to do lately.
Here's the log of  `ueberzugpp layer -o sixel`:
[ueberzugpp-coco.log](https://github.com/jstkdng/ueberzugpp/files/12721748/ueberzugpp-coco.log)
The reason for the `[main] [error] SIGINT received, exiting.` lines is because I pressed `C-c` a few times after running `ueberzugpp`.
I also tried `ueberzugpp layer -o kitty` as well, but got the same result.
